awscli/topics/config-vars.rst

```diff
@@ -0,0 +1,209 @@
+:title: AWS CLI Configuration Variables
+:description: Configuration Variables for the AWS CLI
+:category: General
```

Comment:
The ``return-codes`` topic uses the category ``General Topics``. Do you prefer ``General`` or ``General Topics`` more? We should have some consistency.